impress-org/givewp-next-gen
# Feature: Integrate PayPal Smart Buttons, Hosted Fields

## Description



This PR adds next gen support for the PayPal Donations (PayPal Commerce) gateway, including Smart Buttons and Hosted fields for one-time donations.

## Visuals



![image](https://user-images.githubusercontent.com/10858303/227541636-e4702727-6fd5-461e-92aa-9e628725d6d7.png)

## Testing Instructions



- PayPal button requires a Sandbox account
- Venmo button in Sandbox mode will just fallback to the regular PayPal flow (nothing to see here)
- Hosted fields use the form's regular Donate button
